packages/styles/scss/components/buttongroup/_buttongroup.scss

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../globals/imports';
+
+@mixin buttongroup {
+  $btn-min-width: 26;
+
+  .#{$prefix}--buttongroup {
+    display: flex;
+    flex-direction: column-reverse;
+    margin-top: $layout-01;
+    width: 100%;
+
+    .#{$prefix}--btn {
+      width: 100%;
+      margin-top: $layout-01;
+      min-width: carbon--mini-units($btn-min-width);
+    }
+
+    @include carbon--breakpoint(md) {
+      flex-flow: row-reverse wrap-reverse;
+      justify-content: flex-end;
+      width: 100%;
+
+      .#{$prefix}--btn {
+        width: auto;
+        margin-right: $layout-03;
+        flex-grow: 1;
```

Comment:
So if I'm not mistaken the buttons will always be most times `width: 20rem;` due to the `Button` `max-width`. Is that expected?

Reply:
@wonilsuhibm @oliviaflory @larahanlon2 @annawen1 would you know the answer to this?

Reply:
that shouldn't be the case...it should only be be hit the `20rem max-width` when the text is too long

Reply:
Been playing around with the pure flex way of handling the width growth and it seems to be locked on 20rem. I'll have to play around more with this to see if I can get the behavior to work without resorting to the JS implementation.

Reply:
Issue opened to track this: https://github.com/carbon-design-system/ibm-dotcom-library/issues/559